src/main/java/core/basesyntax/services/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.services;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final String[] colors = new String[]{"red", "blue", "yellow", "green", "rose",

```

Comment:
```suggestion
    private static final String[] COLORS = new String[]{"red", "blue", "yellow", "green", "rose",

```

про нейминг констант можешь почитать тут: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names